ui/overviewTab/overviewTab.pug

```diff
@@ -0,0 +1,107 @@
+        input(name='attr_advantage_athlete_count' type='hidden')
+        div
+          | Natural Athlete
+          | #[span(name='attr_advantage_athlete_count')]
+      .conditional-something
+        input(name='attr_advantage_alert_count' type='hidden')
+        div
+          | Alert
+          | #[span(name='attr_advantage_alert_count')]
+      .conditional-something
+        input(name='attr_advantage_wary_count' type='hidden')
+        div
+          | Wary
+          | #[span(name='attr_advantage_wary_count')]
+      .conditional-something
+        input(name='attr_advantage_mage_count' type='hidden')
+        div
+          | Mage
+          | #[span(name='attr_advantage_mage_count')]
+      .conditional-something
+        input(name='attr_advantage_devout_count' type='hidden')
+        div
+          | Devout
+          | #[span(name='attr_advantage_devout_count')]
+      fieldset.repeating_extraadvantage
+        | #[span.extraadvantagename(name='attr_extraadvantagename')]
+    // Feats
+    .feat-overview.no-add-or-modify-buttons
+      .table-header
+        .featextraordinaryletter Extr
+        .featname Epic Feat
+      fieldset.repeating_feat
+        |
+        | #[span.featextraordinaryletter(name='attr_featextraordinaryletter')]
+        | #[span.featname(name='attr_featname')  ]
+    // Skills
+    .skills-overview.no-add-or-modify-buttons.flex-stack
+      .table-header
+        .skillname Skill
+        .skillability Ability
+        .skillbase Mod
+      fieldset.repeating_skill
+        // This input is used by CSS to hide disciplines
+        input.isdiscipline(name='attr_skillisdiscipline' type='hidden')
+        .skill-overview-row-container
+          |
+          | #[span.skillname(name='attr_skillname')]
+          | #[span.skillability(name='attr_skillability')]
+          input.skillbase(name='attr_skillmodifier' type='number')
+          //
+            These hidden inputs are necessary for the roll to find
+            skillability and skillname
+          input(name='attr_skillability' type='hidden')
+          input(name='attr_skillname' type='hidden')
+          button.skill-roll(name='roll_skillroll' type='roll' value='!EPRoll @{skillability}+@{skillmodifier} Tries @{skillname}: @{skillability} + @{skillmodifier}')
+    = '\n'
+    | &emsp;&emsp;
+    // Weapons
+    .weapons-overview.no-add-or-modify-buttons.flex-stack
+      .table-header
+        .equipmentname Weapon
+        .equipmentcount Count
+        .equipmentdamage Damage
+        .equipmentrange Range
+      fieldset.repeating_weapon
+        .weapons-overview-row-container
+          |
+          | #[span.equipmentname(name='attr_weaponname')]
+          input.equipmentcount(name='attr_weaponcount' type='number' value='1')
+          |
+          | #[span.equipmentdamage(name='attr_weapondamage')]
+          | #[span.equipmentrange(name='attr_weaponrange')]
+        |
+        | #[span.equipmentnotes(name='attr_weaponnotes')]
```

Comment:
I had to put these extra 
```
|
| #[span...]
```
to make `span` elements be their own line.
By default `pug` puts `inline` elements on one line, like:
```
<fieldset...><span class="equipmentnotes" name.../></fieldset>
```

Personally, I'd like three lines...
That being said, we can also not worry about html formatting so much. (or we can post-prettify it)

Reply:
I ended up changing to `beautify-js` to eliminate this pattern.